src/layout/RadioButtons/ControlledRadioGroup.tsx

```diff
@@ -35,16 +34,18 @@ export const ControlledRadioGroup = (props: IControlledRadioGroupProps) => {
     : null;
   const confirmChangeText = langAsString('form_filler.alert_confirm');
 
-  const getLabelPrefixForLikert = () => {
-    if (parentItem?.type === 'Likert' && parentItem.textResourceBindings?.leftColumnHeader) {
-      return `${langAsString(parentItem.textResourceBindings.leftColumnHeader)} `;
-    }
-    return null;
-  };
+  const leftColumnHeader = useNodeItem(node.parent instanceof LayoutNode ? node.parent : undefined, (i) =>
+    i.type === 'Likert' ? i.textResourceBindings?.leftColumnHeader : undefined,
+  );
   const labelText = (
     <LabelContent
       componentId={id}
-      label={[getLabelPrefixForLikert(), langAsString(textResourceBindings?.title)].join(' ')}
+      label={
+        <>
+          {leftColumnHeader ? `${langAsString(leftColumnHeader)} ` : null}
+          <Lang id={textResourceBindings?.title} />
+        </>
+      }
```

Comment:
Is there a reason for using `langAsString` on `leftColumnHeader` instead of `<Lang />`?